src/ol_openedx_canvas_integration/README.rst

```diff
@@ -1,5 +1,69 @@
 ol_openedx_canvas_integration
 =============================
 
+A django app plugin to add ``Canvas Integration`` support in edx-platform.
 
-A plugin that enables canvas integration within edX.
+**NOTE:**
+
+Since the edx-platform’s ``Instructor Dashboard`` does not support the plugin based tabs so we had to make some changes in the edx-platform itself to add ``Canvas`` tab through the plugin.
+
+The ``edx-platform`` branch/tag you're using must include below commit for ``ol-openedx-canvas-integration`` plugin to work properly:
+
+- https://github.com/mitodl/edx-platform/commit/5b8acfceb7fcebab6b5e3aff0493876f439f2b80
+
+
+Installation
+------------
+
+You can install it into any Open edX instance by using any of the following two methods:
+
+- To get the latest stable release from PyPI
+
+.. code-block::
+
+
+    pip install ol-openedx-canvas-integration
+
+
+- To create & install a build locally
+
+.. code-block::
+
+    # Open Terminal
+
+    1. Navigate to "open-edx-plugins" directory
+    2. Run ./pants build  # Do this if you haven’t run it already
+    3. Run ./pants package ::  # This will create a "dist" directory inside "open-edx-plugins" directory with ".whl" & ".tar.gz" format packages for all the "ol_openedx_*" plugins in "open-edx-plugins/src")
+    4. Install the "ol-openedx-canvas-integration" using any of ".whl" or ".tar.gz" files generated in the above step
+
+Dependencies
+------------
+
+Once you’ve installed the plugin you will need set some configuration values as mentioned below.
+
+.. code-block::
+
+    1) Set the value for "CANVAS_ACCESS_TOKEN" into your environment variable  # This can be done through lms.yml or private.py
+    2) Set the value for "CANVAS_BASE_URL"  # This will be the URL of your Canvas instance
+    3) Go to Studio "http://localhost:18010" -> Open a course and navigate to "Advanced Settings" -> Set value for `canvas_course_id`  # This should be the id of a course that exists on Canvas
```

Comment:
This can be more descriptive and readable

```suggestion
**1) edx-platform configuration**

Add the following configuration values to the config file in Open edX. For any release after Juniper, that config file is ``/edx/etc/lms.yml``. These should be added to the top level. **Ask a fellow developer or devops for these values.**

.. code-block::
    
    CANVAS_ACCESS_TOKEN: <some access token value>
    CANVAS_BASE_URL: <the base URL where Canvas is running>

**2) Add course settings value**
1) Open your course in Studio.
2) Navigate to "Advanced Settings".
3) Add a `canvas_course_id` value. This should be the id of a course that exists on Canvas.
```